### ISSUE TITLE
FI Configuration agent / Serveurs

### ISSUE THREAD

**Issue by EC2311** [fusioninventory/fusioninventory-for-glpi#3262]
Bonjour

Nous avons 3 instances serveurs (DEV, PRD et autre instance) : ok

Nous avons donc les agents pour lesquels nous avons configuré leur remontée vers les 3 instances : ok
Nous avons des collectes (à réaliser sur ces agents) : ok

Afin de limiter le lancement des collectes sur les agents ainsi que les inventaires (sachant que les 3 instances sont définies sur les agents), Comment configurer ces collectes et inventaires afin qu'elles ne tournent qu'une seule fois par jour ?
En effet, à l'heure actuelle nous avons du définir les collectes sur les 3 instances, ce qui fait qu'elles s'executent 3 fois alors que l'envoi pourrait s'effectuer vers les 3 serveurs en meme temps...

D'autre part, nous avons sur une instance pres de 5000 agents : ok (en remontée 24 h)
Quand on regarde le menu de base FI (avec l'histogramme) nous voyons pres de 1000 contacts par heure, ce qui voudrait dire qu'au bout de 5 heure ce devrait etre fini, mais on a cela h24...

Merci bien

**Comment by ddurieux** [comment]
Pour la fréqence, c'est en cours de code avec l'agent 3.0 et la prochaine version du plugin fusioninventory ;)

tu as 1000 contacts chaque heure toute la journée?

**Comment by EC2311** [comment]
oui, du moins niveau histogramme. car quand je regarde dans la liste des agents, c'est nettement moins.
peut etre que l'histogramme presente d'autres données en plus ?

cool pour la fréquence 👍

**Comment by ddurieux** [comment]
Non rien de plus, que les inventaires des PC
C'est effectivement bizarre d'en avoir autant toute la journée.
T'es bien en 24h dans la configuration de tes 3 GLPI  pour la fréquence des inventaires?

**Comment by EC2311** [comment]
Sur la prod, comme sur la dev : 
![image](https://user-images.githubusercontent.com/42341169/143837131-2820f6cc-9722-408a-8708-928ad994ab00.png)

niveau DEv, l'histogramme va au plus à 50 (la nuit) -> cohérent
Prod : je reste toujours autour de 1000

**Comment by ddurieux** [comment]
donc c'est pas normal ;(

**Comment by EC2311** [comment]
Je pense que cela doit etre une "erreur" dans l'affichage histogramme
Si je filtre les agents (tout entité confondu) je trouve 192 agents pour 09:00 ce jour
et l'histo m'en presente 1166

pour info (?) 
quand j'affiche cette page : ca mouline sec...
peut etre quelque chose dans la bd qui coince ? 

connais tu la requete ? je vais regarder dans la table agent, je devrais trovuer quelque chose correspondant à 1166 ;)

**Comment by ddurieux** [comment]
C'est là dedans : https://github.com/fusioninventory/fusioninventory-for-glpi/blob/glpi9.5/inc/menu.class.php#L760

**Comment by EC2311** [comment]
En fait le graphe vient de getLastHours et de glpi_plugin_fusioninventory_inventorycomputerstats 
je retrouve donc dans ce fichier pour 09h00 les 1166 que me presente le graphe

inc/inventorycomputerstat.class
-> ligne 93 -> 365 : et le 366 ieme jour des années bissextiles ? ;) 

-> A priori on incremente el compteur via la fonction "increment" ligne 113

**Comment by EC2311** [comment]
euh... je pense avoir trouvé 
j'ai des chiffres pour les jours qui viennent (333, 365...)

donc je me pose la question je n'ai vu que update... mais quand les chiffres de la journée (nouvelle) sont-ils réinitialisés à 0 ?
ou alors il faut ajouter la colonne année en plus ;)

Initialement j'avais supprimé les enreg du jour, mais le quantieme+heure n'est pas créé en absence : donc il faut remettre à 0 le champ counter : par jour :  tache auto ?
